src/client/src/app/components/login/login.component.html

```diff
@@ -0,0 +1,12 @@
+<div class="main-div">
+    <div class="sub-main-div">
+        <div class="form-div">
+            <form>
+                <input type="number" placeholder="Enter Game Pin" formControlName="gamepin" />
```

Comment:
this is a basic login page, so there is going to be **username** and **password** input field, and a **login** button.
Also, 
1. formControlName should be **loginForm**
2. login button should have [disabled]="loginForm.invalid" for making button disabled until the input fields are empty